mycroft/messagebus/client/client.py

```diff
@@ -0,0 +1,216 @@
+# Copyright 2017 Mycroft AI Inc.
```

Comment:
2019?

Reply:
I will respond to your question with a question.  Should I be updating this every time I change a file?

Reply:
What we've done so far is that we've left the copyright at the year of creation / insertion. Which gives a nice sort of history of the project.

Updating the copyright at each change would be tedious to do and maintain and distracting for reviewers imho.